rdr_service/genomic/genomic_job_controller.py

```diff
@@ -1910,6 +1911,32 @@ def check_aw3_ready_missing_files(self):
             )
         self.job_result = GenomicSubProcessResult.SUCCESS
 
+    def check_appointments_gror_changed(self):
+        """ Alerts Color when a participant with scheduled appointment has withdrawn GRoR consent """
+        notification_email_address = config.getSetting(config.GENOMIC_COLOR_PM_EMAIL, default=None)
+        notified_dao = GenomicAppointmentEventNotifiedDao()
+        changed_gror = self.appointment_dao.get_appointments_gror_changed()
+        notified_appointments = []
+        if notification_email_address and changed_gror:
+            body = "Participants with scheduled GC appointments and changed GRoR:\n"
+            body += "participant_id,appointment_id,event_authored_time\n"
+            for appointment in changed_gror:
+                body += f'{appointment.participant_id}, {appointment.appointment_id}, '
+                body += f'{appointment.event_authored_time}\n'
+                notified_appointments.append({
+                    'participant_id': appointment.participant_id,
+                    'created': clock.CLOCK.now(),
+                    'modified': clock.CLOCK.now()
+                })
+            EmailService.send_email(
+                Email(
+                    recipients=[notification_email_address],
```

Comment:
Since we are going to set the notification email as a list in the config, we don't need to add a list cast here.